cmake/FindDependencies.cmake

```diff
@@ -0,0 +1,76 @@
+FIND_PACKAGE(ZLIB REQUIRED)
+IF(ZLIB_FOUND)
+	MESSAGE(STATUS "Looking for Zlib: found")
+ENDIF()
+
+FIND_PACKAGE(Iconv REQUIRED)
+IF(Iconv_FOUND)
+	MESSAGE(STATUS "Looking for iconv: found")
+ENDIF()
+
+IF(USE_OPENAL)
+INCLUDE(FindOpenAL)
+IF(OPENAL_FOUND)
+	MESSAGE(STATUS "Looking for OpenAL: found")
+ELSE()
+	MESSAGE(WARNING "Looking for OpenAL: not found!")
+	MESSAGE(WARNING "If you want to build the OpenAL plugin, get OpenAL from www.openal.org.")
+	MESSAGE(WARNING "If it just wasn't found, try setting the OPENALDIR environment variable.")
+ENDIF()
+ENDIF()
+
+IF(USE_LIBVLC AND NOT MSVC)
```

Comment:
This is worth retesting, since the reason for the MSVC inclusion here was lifted.

Reply:
Not sure what do you mean by that.

Reply:
the "AND NOT MSVC" should not be needed any more. It was added since our vlc use relied on pthreads previously.